ccd-definition/CaseField.json

```diff
@@ -262,17 +262,18 @@
   {
     "LiveFrom": "01/01/2017",
     "CaseTypeID": "CARE_SUPERVISION_EPO",
-    "ID": "standardDirectionsLabel",
-    "Label": "Upload standard directions.",
-    "FieldType": "Label",
+    "ID": "standardDirections",
+    "Label": "1. Standard directions",
+    "FieldType": "StandardDirections",
     "SecurityClassification": "Public"
   },
   {
     "LiveFrom": "01/01/2017",
     "CaseTypeID": "CARE_SUPERVISION_EPO",
-    "ID": "standardDirections",
-    "Label": "Upload a file",
-    "FieldType": "Document",
+    "ID": "otherCourtAdminDocuments",
+    "Label": "Other documents",
+    "FieldType": "Collection",
+    "FieldTypeParameter": "CourtAdminDocuments",
```

Comment:
It should be `CourtAdminDocument` not `CourtAdminDocuments`, see last letter. That complex type represents one document side that ask.